src/compiler/style/css-imports.ts

```diff
@@ -123,9 +123,10 @@ export const getCssImports = async (config: d.Config, compilerCtx: d.CompilerCtx
   styleText = stripCssComments(styleText);
 
   const dir = dirname(filePath);
-  const importeeExt = filePath.split('.').pop().toLowerCase();
+  const importedExt = filePath.split('.').pop().toLowerCase();
```

Comment:
"Importee" is the file being imported, vs. "importer", the file that imports the importee. So the current variable name is correct.

Reply:
Thanks for clarifying, I've removed the commit changing the variable name